.github/workflows/test.yml

```
@@ -12,23 +12,81 @@ jobs:
         strategy:
             fail-fast: false
             matrix:
-                python-version: [3.8, 2.7]
+                plone-version:
+                    - 'Plone51'
+                    - 'Plone52'
+                    - 'Plone60'
+                python-version: [2.7, 3.7, 3.8, 3.9]
+                exclude:
+                    - plone-version: 'Plone51'
+                      python-version: 3.7
+                    - plone-version: 'Plone51'
+                      python-version: 3.8
+                    - plone-version: 'Plone51'
+                      python-version: 3.9
+                    - plone-version: 'Plone52'
+                      python-version: 3.9
+                    - plone-version: 'Plone60'
+                      python-version: 2.7
+                    - plone-version: 'Plone60'
+                      python-version: 3.7
+                    - plone-version: 'Plone60'
+                      python-version: 3.8
```

```suggestion
                    - plone-version: 'Plone60'
                      python-version: 3.7
                    - plone-version: 'Plone60'
                      python-version: 3.8
```
So far Plone 6 supports Python since 3.7 to 3.9